spec/furniture/marketplace/cart_spec.rb

```diff
@@ -22,7 +22,13 @@
       cart.cart_products.create!(product: product_b, quantity: 2)
     end
 
-    it { is_expected.to eql(product_a.price + product_b.price + product_b.price + marketplace.delivery_fee) }
+    it { is_expected.to eql(product_a.price + product_b.price * 2) }
+
+    context "when the #delivery_address is present" do
+      let(:cart) { create(:marketplace_cart, delivery_address: "123", marketplace: marketplace) }
+
+      it { is_expected.to eql(product_a.price + product_b.price + product_b.price + marketplace.delivery_fee) }
```

Comment:
why is `product_b.price` listed twice? Is that a typo or is the test increasing `product_b` quantity in the cart just by adding a 2nd `product_b.price`?

Reply:
Yep, the `cart_product` that references `product_b` has a quantity of `2`; I could do `product_b.price * 2` if that is more clear?

Reply:
IMHO and nitpick: `product_b.price * 2` looks more clear and intentional.

Reply:
Done!